### ISSUE TITLE
allow setting maximum column width (feature req)

### ISSUE THREAD

**Issue by bbrendon** [home-assistant/home-assistant-cli#253]
It would be nice to disable the wrapping of the output when using commands such as `hass-cli state list`.  An option like "max-width" or something similar would be nice. I tried a bunch of the "table-format" options but couldn't find anything to limit the width of the columns and prevent wrapping. 

The main reason for this is the 'state list' output is hard to read with the wrapping.

I suppose I could do something like : `hass-cli  state list | cut -c -170` , but that's too much typing :)

**Comment by maxandersen** [comment]
I agree it's not optimal but not something I have an easy fix for. The tabulate library used to print don't have width/formatting built-in. 

Might need to move to something different to accommodate. Ideas welcome!

**Comment by maxandersen** [comment]
Actually, a hacky fix could be to do an abrupt line cut similar to your cut example and expose as a flag option.

**Comment by bbrendon** [comment]
Probably should be a separate issue but would be nice to maybe use an env var to save some default settings like `export HASSCLI_OPT=` . That way I don't have to type `|sort` and  I can use the built in sort feature easier. Or maybe just change the code to sort the state list by default.

**Comment by bbrendon** [comment]
```
hcli_func() {
    hass-cli   "$@" | cut -c -160 | sort		
}
alias hass-cli=hcli_func
```

**Comment by maxandersen** [comment]
:) do open seperste issue for sorting. That one is fairly trivial to do. 

Support for environment variables and a config file I want too.
